### PR TITLE
update zh-hans translation

### PR DESCRIPTION
**Pull request by ymhierarch**
I added some Chinese translations and corrected some translation errors.
我添加了一些中文翻译并修改了一些翻译错误。

### CODE REVIEW

File: src/assets/i18n/zh-hans.json [hunk 1]
```diff
@@ -148,7 +148,7 @@
   "Choose between a <i>Hex</i> (base 16 numeral) or <i>Passphrase</i> (alphanumeric) style encryption key.": "在 <i>Hex</i>（基数为 16 的数字）或 <i>密码</i>（字母数字）样式的加密密钥之间进行选择。",
   "Choose container": "选择容器",
   "Choose log": "选择日志",
-  "Choose pod": "选择pod",
+  "Choose pod": "选择pod池",
```
Comment (Sagit-chu) [inline src/assets/i18n/zh-hans.json:151]: pod是K8S的概念没有池的意思吧

Comment (ymhierarch) [reply src/assets/i18n/zh-hans.json:151]: 确实，我改一下

**Review by Sagit-chu — APPROVED** [review]

**Comment by Sagit-chu** [comment]
你需要解决冲突
You need to resolve conflicts

**Comment by ymhierarch** [comment]
> 你需要解决冲突
> You need to resolve conflicts

还哪里有问题？

**Comment by ymhierarch** [comment]
Conflict resolved

**Pull request closed without merging**